examples/range-selector.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<head>
+  <meta name="viewport" content="width=device-width">
+  <script type="module" src="../dist/index.js"></script>
+</head>
+<style>
+  /*
+     over-ride the opacity rule so that the media-control-bar is always
+     showing (the default behavior is to hide it when media is playing)
+   */
+  media-control-bar {
+    opacity: 1;
```

Comment:
I've thought about a `<control-bar autohide="numberOfSeconds">`. Looks like that might be useful here if we had an option like `-1` to not hide.